src/backend/postmaster/postmaster.c

```diff
@@ -738,6 +742,8 @@ PostmasterMain(int argc, char *argv[])
 	 */
 	InitializeGUCOptions();
 
+	InitializeTimeouts();
```

Comment:
Seems like `InitializeTimeouts()` [resets all previously active timeouts](https://github.com/arenadata/gpdb/blob/46fb51ca8e4dea8602fca7ffa23b3cb7110b3b33/src/backend/utils/misc/timeout.c#L389-L397). Is this call safe?

Reply:
> Seems like InitializeTimeouts() resets all previously active timeouts. Is this call safe?

Postmaster didn't initialize timers before my change (otherwise I wouldn't need this change).
And this place is before start of other processes (though as far as I see it shouldn't interfere with the timers of other processes).
So yes, I think it is safe to call it here.